new-features/new-feature-template.md

```diff
@@ -0,0 +1,40 @@
+# New Feature Proposal Template
```

Comment:
add documentation/references section.

Add an Assignee

The person responsible for implementing/driving to completion of the blueprint.